Owncloud iOs Client/Tabs/FileTab/FilesViewController.m

```diff
@@ -265,6 +270,15 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
     
+    self.willLayoutSubviews = true;
+    
+    if (IS_IOS8 || IS_IOS9) {
```

Comment:
On all the points that you are using IS_IOS8 || IS_IOS9:
Is it not possible just use !IS_IOS7 (NOT iOS7)? It cloud make easier the change in the future (iOS9)